src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -204,9 +248,40 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
 
                     <div className="patientDetails__main_menu">
                       <h6>{t("patient.usersections")}</h6>
+
                       <div
-                        className="patientDetails__main_menu__item"
-                        onClick={() => setUserSection("exams")}
+                        className={
+                          "patientDetails__main_menu__item " +
+                          isActive("admissions")
+                        }
+                        onClick={() => {
+                          setUserSection("admissions");
+                          setDefaultRoute("/admission");
+                        }}
+                        style={{ display: "flex", alignItems: "center" }}
+                      >
+                        <Assignment
+                          fontSize="small"
+                          style={{ color: "white" }}
+                        />
+                        <span>{t("patient.admissions")}:</span>
+                        <img
+                          src={Arrow}
+                          className="icon_toggle"
+                          alt="Accordion toogle"
+                        />
+                      </div>
+
+                      <div
+                        className={
+                          "align__element patientDetails__main_menu__item " +
+                          isActive("exams")
+                        }
+                        onClick={() => {
+                          setUserSection("exams");
+                          setDefaultRoute("/summary");
+                        }}
+                        style={{ display: "flex", alignItems: "center" }}
```

Comment:
Why are you gonna use the display flex and the class align__element?

I remember that when I release this code the alignaments of both icons and text were right.

Reply:
It is to center the text and the icon if I remember. So if you have some tips to align text with icon, you are welcome!!!!

Reply:
It is to center the text and the icon if I remember. So if you have some tips to align text with icon, you are welcome!!!!

Reply:
![image](https://user-images.githubusercontent.com/8697550/131803570-07d28c62-3dd5-4635-8780-96a667a97720.png)

What I'm saying is that in my local branch the icons are already aligned with the text without any other style or css class.

Reply:
OK, I removed the flex and aligment style. It could be browser incompatibilty.